code/modules/projectiles/updated_projectiles/gun_attachables.dm

```diff
@@ -151,6 +151,7 @@ Defined in conflicts.dm of the #defines folder.
 	G.burst_scatter_mult += burst_scatter_mod
 	G.movement_acc_penalty_mult += movement_acc_penalty_mod
 	G.shell_speed_mod	+= attach_shell_speed_mod
+	G.scope_zoom 		+= scope_zoom_mod
```

Comment:
If its' a boolean why are you adding/subtracting it?

Reply:
Because that's how every other attachment works.